modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -473,6 +474,8 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::checkInputWindow(const GenericReal<is_ad>
       mooseWarning(msg.str());
     else if (behavior == WindowFailure::ERROR)
       mooseError(msg.str());
+    else
+      mooseException(msg.str());
```

Comment:
👍

Reply:
Calling this with `WindowFailure::EXTRAPOLATE` will throw an exception. Wut?

Reply:
no it won't